integritybackend/claim.py

```diff
@@ -398,6 +505,29 @@ def _make_signature_data(self, signatures, meta):
             "starling:signatures": signature_list,
         }
 
+    def _make_signature_data_proofmode(self, pgp_sig, pgp_pubkey, sha256hash, filename):
```

Comment:
This is specific to ProofMode. @makeworld-the-better-one this will potentially have overlap with your "src sig verification" module. I can imagine your impls to contain an func like `generate_starling_c2pa()` or similar.

Reply:
I assume you're referring to #89 (PR #92). For claims I was just using `generate_create`, as the C2PA JSON I needed regarding signatures was already there. Is that a mistake?

Reply:
Discuss later.